---
title: AEP - Unable to connect a to Big Query as a Data Source when adding data
description: Description
solution: Experience Platform
product: Experience Platform
applies-to: Experience Platform
keywords: KCS, AEP, sources, Google, Big Query, datasets
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Christophe Jossic
article-created-date: 5/13/2022 2:36:35 PM
article-published-by: Christophe Jossic
article-published-date: 5/13/2022 2:39:23 PM
version-number: 2
article-number: KA-19525
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=1a607b16-cad2-ec11-a7b5-00224809c27a
exl-id: 14017ad5-9ac9-4932-a596-fabaf45ab773
---
# AEP - Unable to connect a to Big Query as a Data Source when adding data

## Description


We have been trying to connect our Production instance to a Big Query table and we are presented with an error -<b>*"An error occurred getting data from the source"</b>*

Existing dataflows are successfully running.

Adding new data seems the issue.


## Resolution


The issue is in the <b>[!DNL Microsoft] DataFactory API </b>which takes a long time to fetch list of tables if there are a large number of datasets in GBQ project. The fix for this was done by Microsoft but it appears that they released it only for affected customers at the time of the report. Adobe Engineering is currently communicating with Microsoft for the fix to be released for all. If facing this issue from the Adobe Experience Platform perspective, please raise a support ticket with the AEP team.
